products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,380 @@
+WITH collect(player) as players_direct_to_sevilla
+MATCH
+    (t:Transfer)<-[e:TRANSFERRED_IN]-(player:Player)-[:TRANSFERRED_IN]->(:Transfer)<-[:TRANSFERRED_FROM]-(barca:Team)
+WHERE
+    barca.name = "FC Barcelona" AND
+    NOT player IN players_direct_to_sevilla
+WITH player, collect(t) as transfers
+MATCH
+    path_indirect = (a:Team)-[*bfs..10 (e, n | 'Team' IN labels(n) OR ('Transfer' in labels(n) AND n in transfers) )]->(b:Team)
+WHERE
+    a.name = "FC Barcelona" AND
+    b.name = "Sevilla FC"
+UNWIND nodes(path_indirect) as player_path_node
+WITH player_path_node, player
+WHERE 'Team' in labels(player_path_node)
+WITH collect(player_path_node.name) as team_names, player
+RETURN player.name as player_name, team_names
+```
+
+In this query, the only difference is that we need to find players who had a direct transfer to Sevilla first.
+In the next `MATCH` we use that information to check whether players that were transferred from FC Barcelona,
+didn't have direct transfer to Sevilla FC. 
+
+If you are running this in [MemgraphLab](https://memgraph.com/product/lab) with visuals you can change the query
+in some way to get a full graph of two teams, both transfers and players.
+
+```opencypher
+MATCH
+    (player:Player)-[:TRANSFERRED_IN]->(t:Transfer)<-[:TRANSFERRED_FROM]-(barca:Team),
+    (t)-[:TRANSFERRED_TO]->(sevilla:Team)
+WHERE 
+    barca.name="FC Barcelona" AND
+    sevilla.name="Sevilla FC"
+WITH collect(player) as players_direct_to_sevilla
+MATCH
+    (t:Transfer)<-[e:TRANSFERRED_IN]-(player:Player)-[:TRANSFERRED_IN]->(tr:Transfer)<-[:TRANSFERRED_FROM]-(barca:Team)
+WHERE
+    barca.name = "FC Barcelona" AND NOT player  in players_direct_to_sevilla
+WITH player, collect(t) as transfers, collect(e) as connections
+MATCH
+    path_indirect = (a:Team)-[*bfs..10 (e, n | 'Team' IN labels(n) OR ('Transfer' in labels(n) AND n in transfers) )]->(b:Team)
+WHERE
+    a.name = "FC Barcelona" AND
+    b.name = "Sevilla FC"
+UNWIND connections as connection
+RETURN player, connection, path_indirect
+```
+
+The first part of the query where we check whether the player was transferred from "FC Barcelona"
+to "Sevilla FC" stays the same. In the next part where we collect the transfers of those players that were transferred from FC Barcelona,
+we also need to collect those edges `e` between each transfer and player, because MemgraphLab can only draw connections between nodes that we
+return in the last part of the query. And instead of using function UNWIND to return teams as a list, we can just return the variable containing
+the path of the player. If you look closely now at that path variable, you can see that it doesn't contain connections between player and transfer.
+That's the reason why we need to collect those connections between transfer and player in the first place. But we can't just return those connections.
+We need to unwind them and return each connection between `Transfer` and `Player` corresponding to that player's path.
```

Comment:
```suggestion
MemgraphLab graph visual representation draws nodes and edges from query results. If you only have
nodes in the results then only nodes will be drawn on the canvas. If you have both nodes and edges present in the
results, MemgraphLab is able to draw nodes and connections between them because it has all the information relevant
for drawing.

In order to change the query to accommodate that, we need to change the types of results that are returned
and collect any missing edge or node information throughout the query. The first part of the query where we
check whether the player was transferred from "FC Barcelona" to "Sevilla FC" stays the same. In order to draw
all connections from players to transfers, we need to collect edges connecting them. That is the reason why we
are collecting edges `e` through variable `player_to_transfers` because it contains information on which player
is connected to which transfer.
With that in mind, our results contain all the information for the graph visual:
* A path that contains `Transfer` and `Team` nodes, and all the edges collected on the `Team` to `Team` traversal
* A list of `Player` nodes
* A list of `Player - Transfer` edges```

Reply:
:+1: